libcudacxx/include/cuda/std/detail/libcxx/include/__config

```diff
@@ -68,13 +62,10 @@ extern "C++" {
 #  endif // _CCCL_CUDA_COMPILER
 
 #  if _CCCL_COMPILER(CLANG)
-
 // Allow for build-time disabling of unsigned integer sanitization
-#    if !defined(_LIBCUDACXX_DISABLE_UBSAN_UNSIGNED_INTEGER_CHECK) && _CCCL_HAS_ATTRIBUTE(no_sanitize)
-#      define _LIBCUDACXX_DISABLE_UBSAN_UNSIGNED_INTEGER_CHECK \
-        __attribute__((__no_sanitize__("unsigned-integer-overflow")))
+#    if !defined(_LIBCUDACXX_DISABLE_UBSAN_UNSIGNED_INTEGER_CHECK)
+#      define _LIBCUDACXX_DISABLE_UBSAN_UNSIGNED_INTEGER_CHECK _CCCL_NO_SANITIZE("unsigned-integer-overflow")
 #    endif
```

Comment:
I believe we should just replace that macro with `_CCCL_NO_SANITIZE("unsigned-integer-overflow")` because the user should not define a reserved keyword and we are not testing without it